src/app/cli/interpreter.cpp

```diff
@@ -197,7 +197,9 @@ const std::map<std::string, Interpreter::Evaluator> &Interpreter::mEvaluatorMap
 };
 
 const std::map<std::string, std::string> &Interpreter::mUsageMap = *new std::map<std::string, std::string>{
-    {"config", "config get pskc\n"
+    {"config", "config get accesscode\n"
```

Comment:
nit: Thread Group TC call yesterday decided to call this "Thread Administrator Passcode".

Reply:
rename in new patch